sql/glean_clients_daily_v1.sql

```diff
@@ -0,0 +1,116 @@
+CREATE TEMP FUNCTION
+  udf_geo_struct(country STRING,
+    city STRING,
+    geo_subdivision1 STRING,
+    geo_subdivision2 STRING) AS ( IF(country IS NULL
+      OR country = '??',
+      NULL,
+      STRUCT(country,
+        NULLIF(city,
+          '??') AS city,
+        NULLIF(geo_subdivision1,
+          '??') AS geo_subdivision1,
+        NULLIF(geo_subdivision2,
+          '??') AS geo_subdivision2)));
+CREATE TEMP FUNCTION
+  udf_json_mode_last(list ANY TYPE) AS ((
+    SELECT
+      ANY_VALUE(_value)
+    FROM
+      UNNEST(list) AS _value
+    WITH
+    OFFSET
+      AS _offset
+    GROUP BY
+      TO_JSON_STRING(_value)
+    ORDER BY
+      COUNT(_value) DESC,
+      MAX(_offset) DESC
+    LIMIT
+      1));
+CREATE TEMP FUNCTION
+  udf_mode_last(list ANY TYPE) AS ((
+    SELECT
+      _value
+    FROM
+      UNNEST(list) AS _value
+    WITH
+    OFFSET
+      AS
+    _offset
+    GROUP BY
+      _value
+    ORDER BY
+      COUNT(_value) DESC,
+      MAX(_offset) DESC
+    LIMIT
+      1 ));
+--
+WITH
+  base AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    LOWER(client_info.client_id) AS client_id,
+    *
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_fenix.metrics_v1`
+  WHERE
+    client_info.client_id IS NOT NULL ),
+  --
+  numbered_duplicates AS (
+  SELECT
+    *,
+    ROW_NUMBER() OVER (PARTITION BY client_id, submission_date, document_id ORDER BY submission_timestamp) AS _n
+  FROM
+    base ),
+  -- Deduplicating on document_id is necessary to get valid SUM values.
+  deduplicated AS (
+  SELECT
+    * EXCEPT (_n)
+  FROM
+    numbered_duplicates
+  WHERE
+    _n = 1 ),
+  windowed AS (
+  SELECT
+    submission_date,
+    client_id,
+    ROW_NUMBER() OVER w1_unframed AS _n,
+    MIN(SAFE.PARSE_DATE('%F', SUBSTR(client_info.first_run_date, 1, 10))) OVER w1 AS profile_date,
```

Comment:
Is `client_info.first_run_date` roughly equivalent to `profile_date` in core pings, @fbertsch ?

Reply:
Right